module-infrastructure/security/src/main/kotlin/site/yourevents/SecurityConfig.kt

```diff
@@ -31,14 +38,23 @@ class SecurityConfig(
         return http.build()
     }
 
+    @Bean
+    fun webSecurityCustomizer(): (WebSecurity) -> Unit {
+        return { web ->
+            web.ignoring()
+                .requestMatchers("/error", "/favicon.ico")
+        }
+    }
+
     @Bean
     fun corsConfigurationSource(): UrlBasedCorsConfigurationSource {
         val configuration = CorsConfiguration()
         configuration.allowedOrigins = listOf(
             "http://localhost:3000",
             "https://www.yourevents.site",
             "https://yourevents.site",
-            "http://localhost:8080")
+            serverUrl
+        )
```

Comment:
컨벤션을 맞추어서 다음과 같이 변경해주시면 좋을 것 같습니다! 😄 
```suggestion
            serverUrl)
```

Reply:
수정하였습니다!